linter/lintfuncs_test.go

```diff
@@ -21,18 +21,18 @@ func TestLintProducedAtDate(t *testing.T) {
 	}
 
 	t.Run("Old ProducedAt date", func(t *testing.T) {
-		err = LintProducedAtDate(ocspResp)
-		if err == nil {
-			t.Errorf("Should have had error: %s is more than %s in the past", ocspResp.ProducedAt.String(), ProducedAtLimit)
+		status, info := LintProducedAtDate(ocspResp, nil)
+		if status != Failed {
```

Comment:
Are the constants like Error, Passed, Failed from linter.go visible here

Reply:
Yep! Same package `linter`